### ISSUE TITLE
Add ability to switch context using environment variable

### ISSUE THREAD

**Issue by lexuzieel** [digitalocean/doctl#985]
**What is the problem this feature would solve? Please describe.**

It would be really nice to have an ability to change `doctl` configuration for the current session with environment variables, similar to `aws-cli`.

This problem is the most apparent when the user needs to switch between multiple contexts.

**Describe the solution you'd like**

Adding an ability to define an environment variable `DIGITALOCEAN_CONTEXT` as an alternative way to switch context for the current session only:

```bash
$ doctl auth list
default (current)
my-context
$ DIGITALOCEAN_CONTEXT=my-context doctl auth list
default
my-context (current)
```

This will be more in line with `aws-cli` which uses `AWS_DEFAULT_PROFILE` environment variable for the same purpose.

This will also make it much easier to use `doctl` along with the tools such as [direnv](https://direnv.net/)

**Additional context**

**Comment by andrewsomething** [comment]
Hey @lexuzieel,

This should be something that `doctl` already supports.

```
asb@asb-home:~$ DIGITALOCEAN_CONTEXT=terratest doctl auth ls
contract
default
terratest (current)

asb@asb-home:~$ DIGITALOCEAN_CONTEXT=contract doctl auth ls
contract (current)
default
terratest

asb@asb-home:~$ DIGITALOCEAN_CONTEXT=default doctl auth ls
contract
default (current)
terratest 
```

Is it not functioning as expected for you? Maybe we need to look into documenting this better.

**Comment by lexuzieel** [comment]
Yup, sorry - my bad! I actually figured it out just now by looking through code

https://github.com/digitalocean/doctl/blob/48c57d1ba88ded599dcecf911d69b1ce3ba04989/commands/doit.go#L88-L92

In fact now I've found the place in the docs that mentions this exact variable.

I think it might be helpful to designate a dedicated readme section to environment variables? My particular case was that I skipped right into `Configuring Default Values` section while the description of `DIGITALOCEAN_CONTEXT` was at the end of the section above it (I checked the first half of the `Authenticating with DigitalOcean` section).

Either way this is definitely good news and so I can update my powerlike10k prompt segment for `doctl` to act similar to `aws-cli` :)
